readme.md

```diff
@@ -43,46 +46,28 @@
   <a href="https://twitter.com/stenciljs">Twitter</a>
 </h2>
 
-[Stencil](https://stenciljs.com/) is a simple compiler for generating Web Components and static site generated progressive web apps (PWA). Stencil was built by the [Ionic](https://ionic.io/) team for its next generation of performant mobile and desktop Web Components.
-
-Stencil combines the best concepts of the most popular frontend frameworks into a compile-time rather than run-time tool. It combines TypeScript, JSX, an asynchronous rendering pipeline to ensure smooth running animations and lazy-loading, to generate 100% standards-based Web Components that run on both [modern browsers and legacy browsers](https://stenciljs.com/docs/browser-support).
-
-Stencil components are just Web Components, so they work in any major framework or with no framework at all. In many cases, Stencil can be used as a drop in replacement for traditional frontend frameworks given the capabilities now available in the browser, though using it as such is certainly not required.
-
-Stencil also enables a number of key capabilities on top of Web Components, in particular Server Side Rendering (SSR) without the need to run a headless browser, pre-rendering, and objects-as-properties (instead of just strings).
-
-## Getting Started
-
-To create a new project using an interactive cli, run:
-
-```bash
-npm init stencil
-```
-
-To start developing your new Stencil project, run:
-
-```bash
-npm start
-```
-
-## Creating components
-
-Stencil components are TypeScript classes with decorator metadata. The decorators themselves are purely build-time annotations so the compiler can read metadata about each component, and removed entirely for smaller efficient components.
+### Getting Started
 
-Create new components by creating files with a `.tsx` extension, such as `my-component.tsx`, and place them in `src/components`.
+Start a new project by following our quick [Getting Started guide](https://stenciljs.com/docs/getting-started).
+We would love to hear from you!
+If you have any feedback or run into issues using Stencil, please file an [issue](https://github.com/ionic-team/stencil/issues/new) on this repository.
 
+### Examples
+A Stencil component looks a lot like a React component, with the addition of TypeScript decorators:
```

Comment:
```suggestion
A Stencil component looks a lot like a class-based React component, with the addition of TypeScript decorators:
```

possible small clarification since functional components are sort of the default now

Reply:
Ah - good call out!